Also Gradle is REFUSING to build Forge for me so I would appreciate it if you guys could test it for me. Here's the error (which I have NEVER gotten before and should have NOTHING to do with Forge's buildscript):

```
> Task :clean:extractMapped FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':clean:extractMapped' (type 'ExtractZip').
  - Gradle detected a problem with the following location: '/home/jonathing/Development/git/minecraft/minecraft-forge/minecraft-forge/1.20.4/projects/clean/src/main/java'.
    
    Reason: Task ':clean:compileJava' uses this output of task ':clean:extractMapped' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
    
    Possible solutions:
      1. Declare task ':clean:extractMapped' as an input of ':clean:compileJava'.
      2. Declare an explicit dependency on ':clean:extractMapped' from ':clean:compileJava' using Task#dependsOn.
      3. Declare an explicit dependency on ':clean:extractMapped' from ':clean:compileJava' using Task#mustRunAfter.
    
    For more information, please refer to https://docs.gradle.org/8.4/userguide/validation_problems.html#implicit_dependency in the Gradle documentation.
```

--- a/settings.gradle
+++ b/settings.gradle
@@ -26,7 +26,7 @@ dependencyResolutionManagement {
             plugin('apt', 'com.diffplug.eclipse.apt').version('4.2.0')
 
             library('forgespi', 'net.minecraftforge:forgespi:7.1.5') // Needs modlauncher
-            library('modlauncher', 'net.minecraftforge:modlauncher:10.2.2') // Needs securemodules
+            library('modlauncher', 'net.minecraftforge:modlauncher:10.2.3') // Needs securemodules
             library('securemodules', 'net.minecraftforge:securemodules:2.2.21') // Needs unsafe
             library('unsafe', 'net.minecraftforge:unsafe:0.9.2')
             library('accesstransformers', 'net.minecraftforge:accesstransformers:8.2.1')

--- a/settings.gradle
+++ b/settings.gradle
@@ -27,7 +27,7 @@ dependencyResolutionManagement {
 
             library('forgespi', 'net.minecraftforge:forgespi:7.1.5') // Needs modlauncher
             library('modlauncher', 'net.minecraftforge:modlauncher:10.2.2') // Needs securemodules
-            library('securemodules', 'net.minecraftforge:securemodules:2.2.20') // Needs unsafe
+            library('securemodules', 'net.minecraftforge:securemodules:2.2.21') // Needs unsafe
             library('unsafe', 'net.minecraftforge:unsafe:0.9.2')
             library('accesstransformers', 'net.minecraftforge:accesstransformers:8.2.0')
             library('coremods', 'net.minecraftforge:coremods:5.2.4')

--- a/settings.gradle
+++ b/settings.gradle
@@ -45,7 +45,7 @@ dependencyResolutionManagement {
             library('jimfs', 'com.google.jimfs:jimfs:1.3.0')
             bundle('jimfs', ['guava', 'failureaccess'])
 
-            version('bootstrap', '2.1.7')
+            version('bootstrap', '2.1.8')
             library('bootstrap',      'net.minecraftforge', 'bootstrap'     ).versionRef('bootstrap') // Needs modlauncher
             library('bootstrap-api',  'net.minecraftforge', 'bootstrap-api' ).versionRef('bootstrap')
             library('bootstrap-dev',  'net.minecraftforge', 'bootstrap-dev' ).versionRef('bootstrap')

